core/main_test.go

```diff
@@ -418,6 +419,21 @@ func ExampleRun_node_db() {
 	//    --help, -h  show help
 }
 
+func ExampleRun_node_profile() {
+	run("node", "profile", "--help")
+	// Output:
+	// 	NAME:
+	// 	chainlink node profile - Collect profile metrics from the node.
+
+	//  USAGE:
+	// 	chainlink node profile [command options] [arguments...]
+
+	//  OPTIONS:
+	// 	--seconds value, -s value     duration of profile capture (default: 8)
+	// 	--output_dir value, -o value  output directory of the captured profile (default: "/tmp/")
+
+}
```

Comment:
IIRC the Output comments block has to be contiguous through the end of the func to actually be executed a testable example.
```suggestion
	// Output:
	// 	NAME:
	// 	chainlink node profile - Collect profile metrics from the node.
	//
	//  USAGE:
	// 	chainlink node profile [command options] [arguments...]
	//
	//  OPTIONS:
	// 	--seconds value, -s value     duration of profile capture (default: 8)
	// 	--output_dir value, -o value  output directory of the captured profile (default: "/tmp/")
}
```

Reply:
Yeah fixed that. It was ignoring the block and passing regardless before